### PR TITLE
Update changelog for sprint 47

### PR DESCRIPTION
**Pull request by bgfoshay**
## What’s changing?

Update the changelog for sprint 47

## Why?

Update the changelog for sprint 47

## This PR has:

- [ ] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

**Review by ccaprio97 — APPROVED** [review]
Nitpick: Search reason was in current sprint
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@ All notable changes to this project will be documented in this file.
...
+## [1.1.1.47] - 2022-08-09
+
+### Changed
+- Parameterized IAC to provide APIM SKU based on environment
+- Server parameters for Postgres in the Postgres ARM templates to increase logging
+
+### Added
+- Unauthorized banner to Participant Search, Match Search, Match Detail, and Match List pages. 
+- Support for tests that render & test CSHTML pages  
+- Script that generates a dependency report containing a list of .Net and JS dependencies
```
Comment (dborgen2) [inline CHANGELOG.md:15]: There was also a new Unauthorized Page for when a user doesn't have a location or a role claim

**Review by ryangeorgeharvey — COMMENTED** [review]
I would add the changes for Log Analytics workspace and Application Insights for Functions. Steal from here: https://github.com/18F/piipan/pull/3658

**Review by ryangeorgeharvey — APPROVED** [review]
LGTM

**Pull request merged**